main.go

```diff
@@ -507,29 +233,26 @@ func main() {
 
 	dockerAuth := getDockerAuthConfig(config.DockerRepository)
 	scExec := newSidecarExecutor(dockerClient, &dockerAuth, config)
-	dconfig := executor.DriverConfig{Executor: scExec}
 
-	driver, err := executor.NewMesosExecutorDriver(dconfig)
-	if err != nil || driver == nil {
-		log.Fatalf("Unable to create an ExecutorDriver: %s", err)
+	// The Mesos lib has its own env configuration, so load that up as well.
+	// This supports all the MESOS_* env vars passed by the agent on startup.
+	cfg, err := mesosconfig.FromEnv()
```

Comment:
The newer Mesos lib takes its config from env vars and not cli args. This is nice, but means there might be a change or two in `executor.sh` that is required.